src/components/DataGroupPopup.js

```diff
@@ -25,6 +27,27 @@ const DataGroupPopup = ({ object, type, closeDescription }) => {
         axios.get(`${constants.ROOT_URL}/api/user/maps/`, getAuthHeader())
             .then((response) => {
                 dispatch({ type: 'POPULATE_MY_MAPS', payload: response.data });
+
+                if (selectedMap.map.eid === currentMapId) {
+                    console.log("saving to currently open map, reloading");
+                    const currentMap = response.data.find(map => map.map.eid === selectedMap.map.eid);
+                    const savedMap = JSON.parse(currentMap.map.data);
+                    if (savedMap.mapLayers.activeLayers) {
+                        savedMap.mapLayers.landDataLayers = savedMap.mapLayers.activeLayers;
+                    }
+                    //fix that some have no dataLayers
+                    if (!savedMap.mapLayers.myDataLayers) {
+                        savedMap.mapLayers.myDataLayers = [];
+                    }
+                    savedMap.currentLngLat = lngLat;
+                    savedMap.currentZoom = zoom;
+                    dispatch({
+                        type: 'LOAD_MAP_STATIONARY',
```

Comment:
Won't this remove any unsaved changes (e.g. drawn objects, layers that have been hidden/shown) if we reload the current map? I think we'll be able to do this after we've added autosave functionality across the whole app.
I might be mistaken though since I'm just doing a code read and haven't tested.

Reply:
you're right, but without this, if you have the map open and you add a datagroup item to it, it doesn't do anything at all.

I'll ask Lynne tomorrow what to have in the meantime.

Reply:
I've added the autosave fix that we talked about for this in PR #183, ready to merge after the release is done.